core/io/multiplayer_api.h

```diff
@@ -168,7 +168,7 @@ class MultiplayerAPI : public Reference {
 	MultiplayerAPI();
 	~MultiplayerAPI();
 
-	static const int LOCAL_CLIENT_SENDER_ID = INT_MAX - 1;
+	static const int LOCAL_CLIENT_SENDER_ID = 2147483646; // INT_MAX - 1
```

Comment:
That's weird. Maybe we'd need to include `<limits.h>` or something.

Reply:
Updated with std::numeric_limits